src/App.tsx

```diff
@@ -1,14 +1,51 @@
-/* eslint-disable max-len */
-import React from 'react';
-import 'bulma/css/bulma.css';
-import '@fortawesome/fontawesome-free/css/all.css';
+import {
+  FC,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 
-import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+
+import { TodoContext } from './context/ContextTodo';
+
+import { TodoFilter, getPreparedTodos } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { TodoList } from './components/TodoList';
 import { Loader } from './components/Loader';
 
-export const App: React.FC = () => {
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+export const App: FC = () => {
+  const {
+    setTodos,
+    isModalOpen,
+    todos,
+    inputField,
+    filteredBy,
+  } = useContext(TodoContext);
+
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Better to set `false` as a default value and change it to `true` before API call

Reply:
Not fixed

Reply:
I don't think it matters when you setting loader in one place

Reply:
@FrontKid This is a rule of good code - by default, exclude what we don't need always and include only when we have to use it.